tsconfig.json

```diff
@@ -1,21 +1,21 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
+    "types": ["vite/client", "node"],
+    "allowJs": false,
+    "skipLibCheck": false,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "es2020",
+    "moduleResolution": "Node",
```

Comment:
what's the difference between a module resolution of `node` and `Node` ?

Reply:
I think there's no difference, I changed `node` for `Node` because it's write `Node` in [documentation](https://www.typescriptlang.org/docs/handbook/module-resolution.html)